compiler+runtime/include/cpp/jank/read/lex.hpp

```diff
@@ -47,6 +47,15 @@ namespace jank::read::lex
     /* Has string data. */
     escaped_string,
     eof,
+    /* Has two integer data. */
+    ratio,
```

Comment:
Please put this *before* `eof`; the intention is that it's the last one. Would seem odd to have it in the middle of our token types. Since it's numeric, I think it makes sense to go right after `real`.